### PR TITLE
feat: Add commit query in service layer

### PR DESCRIPTION
**Pull request by talwinder50**
Closes:#33
Signed-off-by: talwinder.kaur <talwinder.kaur@securekey.com>

### CODE REVIEW

File: internal/server/collection/collection_test.go [hunk 1]
```diff
@@ -65,7 +65,50 @@ func TestWriteRequestError(t *testing.T) {
...
+	serverConfig := server.Config{PageSize: pageSize, CollectionStore: collectionStore}
+	oid := "7e181b7ca4b04246bcc064eede4af26c"
+	commitQueryReq := newCommitQueryRequest(oid,"")
+
+	response, err := ServiceRequest(&serverConfig, commitQueryReq)
+	require.Nil(t, err)
+	assert.NotNil(t, response)
+	assert.Equal(t, difContext, response.CommitQueryResponse.AtContextField)
+}
+func TestCommitQueryError(t *testing.T) {
```
Comment (llorllale) [inline internal/server/collection/collection_test.go]: @talwinder50 let's add a single blank line between the definition of all these functions

Comment (rolsonquadras) [reply internal/server/collection/collection_test.go]: @talwinder50 looks like Go formatter not applied/enabled in the IDE.

Comment (talwinder50) [reply internal/server/collection/collection_test.go]: done

Comment (llorllale) [reply internal/server/collection/collection_test.go]: @talwinder50 what I meant was to introduce a new blank line between the closing curly brace of a function and the declaration of the next function. In this file and in the others that are part of this change.

Comment (talwinder50) [reply internal/server/collection/collection_test.go]: @llorllale go-land formatter doesnt add these lines , just curious why we need to add specifically ??

Comment (talwinder50) [reply internal/server/collection/collection_test.go]: done

**Pull request merged**